marlowe-playground-client/README.md

```diff
@@ -2,25 +2,24 @@
 
 ## Getting started
 
+**Starting the backend server**
+
 Make sure you have a local backend server running first:
 ```bash
-$(nix-build -A marlowe-playground.server-invoker)/bin/marlowe-playground webserver
+marlowe-playground-server
 ```
 
 Check the [backend documentation](../marlowe-playground-server/README.md) for more information on how to setup the Github OAuth application.
 
-Now we will build and run the front end:
+**Starting the frontend server**
+
+You can build and start the server with one simple command:
 ```bash
-cd marlowe-playground-client
-# Generate the purescript bridge files
-marlowe-playground-generate-purs
-# Download javascript dependencies
-npm install
-# Install purescript depdendencies
-npm run purs:compile
+npm run webpack:server
```

Comment:
`marlowe-playground-server` needs to be executed in the main folder, but this should be executed inside the `marlowe-playground-client` folder, so I'd leave the `cd marlowe-playground-client` command.